contents/blog/clickhouse-vs-snowflake.md

```diff
@@ -0,0 +1,146 @@
+
+However, the biggest difference is in Clickhouse’s strength. Let’s return to the analogy of ClickHouse being a jet ski. ClickHouse is fast. Incredibly, unbelievably fast. In particular, ClickHouse can return complex aggregations of terabyte-level data spanning millions of rows in milliseconds. In comparison, Snowflake takes dozens of seconds to query gigabyte-level data. 
+
+ClickHouse accomplishes this by optimizing the database for speed at returning aggregates. ClickHouse isn’t designed to be utilized like a stock PostgreSQL or mySQL database; it’s particularly good at write-heavy, mutation-low, read-and-reduce-heavy operations. It's perfect solution for analytics providers, like [PostHog](https://posthog.com/).
+
+However, normal on-prem ClickHouse lacks a lot of features that Snowflake likely considers table-stakes: 
+
+- A dedicated non-technical-friendly GUI for exploring and visualizing data
+- The ability to modify entry data without massive performance burns
+- The ability to scale architecture seamlessly with no migrations whatsoever
+
+### ClickHouse Cloud and Altinity Cloud
+
+It is slightly silly to compare Snowflake and ClickHouse without mentioning ClickHouse Cloud and Altinity Cloud. Both solutions make deploying and maintaining a ClickHouse instance easy, similar to the core value prop of Snowflake’s feature-loaded GUI. In particular, ClickHouse Cloud and Altinity Cloud address the final bullet in the previous section; they make scaling architecture seamless without frustrating data migrations.
+
+ClickHouse Cloud – a new product launched in 2022 by ClickHouse Inc (ClickHouse’s primary contributor) – is exclusively available on AWS, with plans to support both Google Cloud and Azure like Snowflake. Meanwhile, Altinity Cloud – built by Altinity Inc (also contributors to ClickHouse) – has support for both AWS and Google Cloud. Altinity Cloud is also Kubernetes-compatible. 
+
+The big difference between ClickHouse Cloud and Altinity Cloud is how they store data. ClickHouse Cloud is engineered like Google BigQuery. The data is stored in object storage. Altinity Cloud meanwhile runs ClickHouse similar to how an on-prem or local instance works; virtually anything you can do on a local instance is possible on Altinity Cloud. 
+
+Separately, an advantage of ClickHouse Cloud is that it includes some exclusive features not available in the ClickHouse core distribution, including a SQL GUI explorer with some of Snowflake’s basic data exploration features. 
+
+Overall, ClickHouse Cloud or Altinity Cloud are excellent options for companies that want a managed version of ClickHouse on the public cloud. Their emergence sharply closes the gap between ClickHouse and Snowflake. 
+
+### Who uses Snowflake and ClickHouse?
```

Comment:
This section should be moved before the conclusion.

Reply:
Good call